.github/workflows/nightly-tests.yml

```diff
@@ -157,10 +157,10 @@ jobs:
           rm -rf results
           mkdir -p results
 
-      - name: Run guest benchmarks
-        run: |
-          source .venv/bin/activate
-          bash ./openvm/scripts/run_guest_benches.sh
+      #- name: Run guest benchmarks
+      #  run: |
+      #    source .venv/bin/activate
+      #    bash ./openvm/scripts/run_guest_benches.sh
```

Comment:
Doing a quick test without these and above to see if it works, then will bring these back